app/scripts/lib/transaction/util.ts

```diff
@@ -111,7 +112,16 @@ export async function addTransaction(
 
       const securityAlertResponse = await ppomController.usePPOM(
         async (ppom) => {
-          return ppom.validateJsonRpc(ppomRequest);
+          try {
+            return ppom.validateJsonRpc(ppomRequest);
+          } catch (error) {
+            // send to sentry
+            captureException(error);
+            return {
+              reason: error,
+              result_type: BlockaidResultType.Errored,
+            };
+          }
```

Comment:
Can you plz check with product team is its ok to have a value for reason apart from those predefined for it.

Reply:
I've confirmed it's with Olu :) thank you for suggesting this check!

Reply:
I spoke with @bschorchit she said it's fine to have the error in the reason.

This is the description/proposed solution in the issue

```
We're seeing ppom throwing errors on Flask, but we don't have further details on what the error is. We should capture more details about the error.

As discussed previously we can leverage either Sentry or Mixpanel for that. If we leverage Sentry, we should make sure that these errors do appear on Sentry and are not capped out due to Sentry hitting it's error limit. If we choose Mixpanel, we can pass the error as the value for security_alert_reason property or add a new property to the events to pass the error. We could also do both Sentry and Mixpanel.
```